### ISSUE TITLE
compile with FreeRDP support failed on Funtoo Linux

### ISSUE THREAD

**Issue by ofwar** [FreeRDP/Remmina#526]
When i want to compile remmina with FreeRDP support, that failed.
I have open a bug on bugs.funtoo.org cf. https://bugs.funtoo.org/browse/FL-2097

Here's the compile log: http://pastebin.com/vKUaqG03

**Comment by antenore** [comment]
Hi @ofwar 

This is not related or caused by remmina.

Your are missing freerdb on your machine and probably other packages as well.

I'm closing this as "not a bug"

**Comment by ofwar** [comment]
Hi @antenore, 

A bug from FreeRDP package so ?
FreeRDP is present on my system: 
*  net-misc/freerdp
      Latest version available: 1.2.0_beta1_pre20141115
      Latest version installed: 1.1.0_beta1_p20130710-r2
I have compile all version available on my Funtoo package manager...

I have also tested all Remmina version, also the Git version...

**Comment by giox069** [comment]
According to this error:
```
/var/tmp/portage/net-misc/remmina-1.1.1/work/Remmina-1.1.1/remmina-plugins/rdp/rdp_plugin.c:260:3: erreur: too few arguments to function ‘rfx_context_new’
```
you are trying to compile Remmina against the wrong freerdp version. Check the libraries installed in your system. Maybe you have libraries files of freerdp 1.0 installed.
